Laravel.gitignore

```diff
@@ -8,9 +8,14 @@ app/storage/
 # Laravel 5 & Lumen specific
 public/storage
 storage/*.key
+Homestead.json
+Homestead.yaml
 .env.*.php
 .env.php
 .env
 
+# Logs
```

Comment:
I made this section to any log files created by system that shouldn't be published.
npm-debug.log for example, created by node.
That file can comes maybe because there was error in laravel-elixir installation.
but it's not related to the app (laravel project).